definitions/npm/redux-thunk_v2.x.x/flow_v0.83.x-/redux-thunk_v2.x.x.js

```diff
@@ -0,0 +1,29 @@
+// Types from redux with generics removed to be compatible
+// with redux & redux-mock-store,
```

Comment:
```suggestion
// with redux-thunk & redux-mock-store,
```

?

Reply:
That's were we have a problem, why would the redux types shouldn't be "compatible"?

Reply:
What I mean here is that because `Middleware` is typed from `@@redux` it should have compatibility when passed into redux or redux-mock-store. Because these other libs have some type requirements when passing into them.

I guess maybe these comments could straight up be removed? Is that ok with you?

Reply:
Sorry but I still don't get it :sweat_smile: 
Why wouldn't we copy/paste the required types from the "real" redux libdef?

Reply:
I've updated the comments. Let me know if it's more clear, it could do without comments but I felt it's better to write something so that we're aware of future state

Reply:
Actually my question is why do the types here differs from the ones in [the redux libdef](https://github.com/flow-typed/flow-typed/blob/c53229884877b9116a985a202c6cc966149f0e0e/definitions/npm/redux_v4.x.x/flow_v0.134.x-/redux_v4.x.x.js#L10-L20)? Mostly removed generics.

Reply:
I guess I was just lazy. But I get your point. Made an update anyways now so that it's matching redux!